data/strings/strings.txt

```diff
@@ -23454,3 +23454,42 @@
     vi = Dấu chân
     zh-Hans = 追踪
     zh-Hant = 追踪
+
+  [splash_subtitle]
+    comment = OpenStreetMap text on splash screen
+    tags = android,ios
```

Comment:
Is it used on ios?

Reply:
ah i'll remove that for now, i'd like to add it to iOS too but i'll do it once android is done.